schema/schema.py

```diff
@@ -22,20 +30,22 @@ def format_param(param: click.core.Option) -> Dict[str, Union[Optional[str], Lis
                        a list of those allowed options
     * default - The default option for that parameter
     """
-    formatted_param: Dict[str, Union[Optional[str], List[str]]] = {"name": param.name, "help": param.help}
+    formatted_param: Dict[str, Any] = {"title": param.name, "description": param.help}
```

Comment:
It was done partly for linting, but mostly just to improve readability, since I realized the value of those keys could be more than just strings populating that field, especially for the `"default"` key (i.e. `local start-api --port` has a default value of `3000` as an integer, `sync --dependency-layer` has a default value of `True` as a boolean). To include every possible data type would become more verbose than just saying `Any`, and would also make the linter throw errors if we were to introduce additional default values for other parameter types (e.g. if we added a default `number` parameter, which would then need to be included)